easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-1.12.1-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-bundle'
+version = '1.12.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://pytorch.org/'
+description = """PyTorch with compatible versions of official Torch extensions."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('RE2', '2022-06-01'),  # for torchtext
+]
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('PyTorch', version, versionsuffix),
+    ('Pillow-SIMD', '9.2.0'),  # for torchvision
+    ('SentencePiece', '0.1.97'),  # for torchtext
+    ('tqdm', '4.64.0'),  # for torchtext
+    ('double-conversion', '3.2.0'),  # for torchtext
+]
+
+use_pip = True
+
+exts_list = [
+    ('torchvision', '0.14.1', {
+        'source_urls': ['https://github.com/pytorch/vision/archive'],
+        'sources': [{'download_filename': 'v0.14.1.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['ced67e1cf1f97e168cdf271851a4d0b6d382ab7936e7bcbb39aaa87239c324b6'],
+    }),
+    ('torchtext', '0.14.1', {
```

Comment:
This won't work without additional effort I think, at least not if we want to be sure that the provided `SentencePiece` is used.

@PetrKralCZ has been fighting with this, and is about to open a PR for a `torchtext` easyconfig...

Reply:
ah yes, i forgot about that sed thing while i was fighting the download filenames